selenium/src/test/java/au/org/emii/portal/tests/ImosLinkCheckTest.java

```diff
@@ -0,0 +1,38 @@
+package au.org.emii.portal.tests;
+
+import au.org.emii.portal.utils.SeleniumUtil;
+import au.org.emii.portal.utils.WebElementUtil;
+import org.apache.log4j.Logger;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+//Link to IMOS website
+public class ImosLinkCheckTest extends BaseTest {
+
+    private static Logger log = Logger.getLogger(ImosLinkCheckTest.class.getName());
+
+    //Click on the IMOS link at the top right of the screen (22/08 now at bottom of screen)
+    @Test(invocationCount = invocationCount, threadPoolSize = threadPoolSize)
+    public void ImosLinkCheckTest() {
```

Comment:
Again named like a constructor but is a method.  java method should start with a lower case letter and should not be named the same as the class